pack/angel.json

```diff
@@ -86,7 +86,7 @@
     {
         "attack": 1,
         "attack_cost": 1,
-        "attack_text": "<b>Hero Response</b>: After Psylocke attacks, if you are:\n\u2022 Angel, heal 1 damage from Psylocke.\n\u2022 Archangel, ready your hero.",
+        "attack_star": true,
```

Comment:
Because `attack_star` and `thwart_star` haven't officially been added yet, I think we should not include those boolean properties on this PR until that's added on the backend and documented on the frontend, in case those properties end up getting renamed during the code review process.

We can add them in once those properties are part of the process.